source/features/highest-rated-comment.tsx

```diff
@@ -60,10 +60,9 @@ function highlightBestComment(bestComment: Element): void {
 }
 
 function linkBestComment(bestComment: HTMLElement): void {
-	// Find position of comment in thread
-	const position = select.all(commentSelector).indexOf(bestComment);
+	const firstTimelineItem = select('#js-timeline-progressive-loader')!.nextElementSibling!;
 	// Only link to it if it doesn't already appear at the top of the conversation
-	if (position < 3) {
+	if (firstTimelineItem === bestComment) {
```

Comment:
I think this brings the limit much higher. The "position" starts at the second \*comment\* I think, while this is the first comment, right?

Reply:
This is explained in the PR description.

The problem is the first comment doesn't necessarily appear at the "top of the conversation".

Reply:
The logic remains, whatever counts as position (events, etc) this will effectively show a link to the second comment in a thread, potentially only 100px lower, won't it? That's not a useful link.